Gemfile

```diff
@@ -90,7 +90,7 @@ gem 'cancancan', '~> 1.12.0', '< 1.13.0'
 gem 'charlock_holmes', '~> 0.7.0'
 gem 'dalli', '~> 2.7.0'
 gem 'dynamic_form', '~> 1.1.0'
-gem 'exception_notification', '~> 4.0.0', '< 4.1.0'
+gem 'exception_notification', '~> 4.1.0', '< 4.1.2'
```

Comment:
https://github.com/mysociety/alaveteli/commit/cb9b2a098fc7334d49d963092569d8ec42f6cf2b originally mentioned the existence of a bug.